lib/mutant/mutator/node/send.rb

```diff
@@ -18,6 +18,8 @@ class Send < self
           reverse_each:  %i[each],
           reverse_merge: %i[merge],
           map:           %i[each],
+          sample:        %i[first last],
+          first:         %i[last],
```

Comment:
I considered it, but in the case where one actually doesn't care which item is selected out of a list, it seemed like having a stable state would be nice. Based on the conversation I incited, I think you guys have thought about this significantly more than I have though. I'd definitely add last -> first and shift to first, pop to last, if it were to help the PR along.